cmd/csi/provisioner/cmd.go

```diff
@@ -0,0 +1,119 @@
+package provisioner
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/cmd/config"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/scheme"
+	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
+	csiprovisioner "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi/provisioner"
+	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/env"
+	"github.com/Dynatrace/dynatrace-operator/pkg/version"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
+)
+
+const (
+	use = "csi-provisioner"
+
+	metricsBindAddress   = ":8090"
+	defaultProbeAddress  = ":8091"
+	livenessEndpointName = "/livez"
+	livezEndpointName    = "livez"
+)
+
+var probeAddress string
+
+func New() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  use,
+		RunE: run(),
+	}
+
+	addFlags(cmd)
+
+	return cmd
+}
+
+func addFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&probeAddress, "health-probe-bind-address", ":10090", "The address the probe endpoint binds to.")
+}
+
+func run() func(*cobra.Command, []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		unix.Umask(dtcsi.UnixUmask)
+		version.LogVersion()
+		logd.LogBaseLoggerSettings()
+
+		kubeConfig, err := config.NewKubeConfigProvider().GetConfig()
+		if err != nil {
+			return err
+		}
+
+		csiManager, err := createManager(kubeConfig, env.DefaultNamespace())
+		if err != nil {
+			return err
+		}
+
+		signalHandler := ctrl.SetupSignalHandler()
+
+		err = createCsiDataPath(afero.NewOsFs())
+		if err != nil {
+			return err
+		}
+
+		err = csiprovisioner.NewOneAgentProvisioner(csiManager, createCsiOptions()).SetupWithManager(csiManager)
+		if err != nil {
+			return err
+		}
+
+		err = csiManager.Start(signalHandler)
+
+		return errors.WithStack(err)
+	}
+}
+
+func createCsiDataPath(fs afero.Fs) error {
+	return errors.WithStack(fs.MkdirAll(dtcsi.DataPath, 0770))
```

Comment:
could you please extract it to constant? 
`readWriteExecutePermission = 0770`

Reply:
done